bundle/Resources/config/slot.yml

```diff
@@ -0,0 +1,46 @@
+services:
+    Netgen\TagsBundle\Core\SignalSlot\AbstractPublishSlot:
```

Comment:
Please use the old notation of services with explicit class declaration. This bundle still supports Symfony 2.8.

Also, add `public: false` to all new services.